src/natcap/invest/ui/model.py

```diff
@@ -1961,7 +1961,21 @@ def save_to_python(self, filepath=None):
         # Generated by InVEST {invest_version} on {today}
         # Model: {modelname}
 
+        import logging
+        import sys
+
         import {py_model}
+        import natcap.invest.utils
+
+        LOGGER = logging.getLogger(__name__)
+        root_logger = logging.getLogger()
+
+        handler = logging.StreamHandler(sys.stderr)
```

Comment:
This looks great. Just one small suggestion to change the stream to `sys.stdout`. Either stream works, but I think it's probably more conventional to output logging to stdout rather than stderr. That way someone that tries to redirect the log to a file with `python python_script.py > logfile.txt` (as opposed to `&>`) will get the output.

Reply:
Okay, makes sense! Just changed it